src/ensureRefIsUsedByChild.js

```diff
@@ -0,0 +1,19 @@
+import isDOMElement from './isDOMElement';
+
+export const errorMessage =
+  '<Waypoint> needs a DOM element to compute boundaries. The child you passed is neither a ' +
+  'DOM element (e.g. <div>) nor does it uses innerRef prop.\n\n' +
```

Comment:
"nor does it uses innerRef prop" could be "nor does it use the `innerRef` prop".